src/components/sectionSelector/DropDown.js

```diff
@@ -31,9 +31,7 @@ export default ({
           </option>
         ))}
       </select>
-      {errors && errors.type === 'required' && (
-        <p className="errorMsg">Полето е задължително.</p>
-      )}
+      <p style={{ color: 'red' }}>{errors?.message}</p>
```

Comment:
```suggestion
      {errors && <p style={{ color: 'red' }}>{errors.message}</p>}
```
така няма да имаме излишни параграфи, които могат да влияят на пространството.